selfdrive/car/gm/carstate.py

```diff
@@ -54,11 +55,12 @@ def update(self, pt_cp, cam_cp, loopback_cp):
     # https://static.nhtsa.gov/odi/tsbs/2017/MC-10137629-9999.pdf
     ret.brake = pt_cp.vl["ECMAcceleratorPos"]["BrakePedalPos"]
     if self.CP.networkLocation != NetworkLocation.fwdCamera:
+      # TODO: verify 17 Volt can enable for the first time at a stop and set brake_pressed
       ret.brakePressed = ret.brake >= 8
```

Comment:
17 Volt faults if enabled under 18 mph the first time. We either need to handle it properly, or enabling at 0 mph doesn't fault. Left out ASCM GM to avoid allowing any faults with a TODO